src/3_branch.js

```diff
@@ -285,15 +286,9 @@ Branch.prototype['init'] = function(app_id, options, callback) {
  * immediately, otherwise, it will return once Branch has been initialized.
  * ___
  */
-Branch.prototype['data'] = function(callback) {
-	if (!callback) { return; }
-
-	var self = this;
-	this._queue(function(next) {
-		callback(null, utils.whiteListSessionData(utils.readStore(self._storage)));
-		next();
-	});
-};
+Branch.prototype['data'] = wrap(2, function(done) {
```

Comment:
let's also use the constants here -- if you put callback_params outside of wrap they can be accessed here, it's much clearer to read with a constant